AllASPFuncs.py

```diff
@@ -659,16 +659,24 @@ def find_parq(ID, path = '/hpc/group/cosmology/OpenUniverse2024/roman_rubin_cats
     Find the parquet file that contains a given supernova ID.
     '''
     files = os.listdir(path)
+
+    star = True if len(str(ID)) == 11 else False
+    
     if star:
         files = [f for f in files if 'pointsource' in f]
         files = [f for f in files if 'flux' not in f]
     else:
         files = [f for f in files if 'snana' in f]
     files = [f for f in files if '.parquet' in f]
+
```

Comment:
Not for this PR, but I would suggest using an if statement to construct the regex.  Then make the actual operation that does something one line and general.  Also consider just using a `glob` in the first place to get the file list instead of `os.listdir`.